### PR TITLE
Add post "The Pain and the Joy of creating isomorphic apps in ReactJS"

### PR DESCRIPTION
**Pull request by koorchik**

### CODE REVIEW

File: posts/2015-12-12-isomorphic-react-in-real-life.md [hunk 1]
```diff
@@ -0,0 +1,535 @@
...
+
+<!--more-->
+
+Ths post originally appeared on the [koorchik's blog](http://blog.koorchik.com/isomorphic-react/).
+
+## Preface
+
+Let's start with the term "Isomorphic application". An isomorphic application is just a regular single page application that can be run on server. I do not like the word "isomorphic", I believe that programmers should struggle complexity (not only in code) and the word "isomorphic" сomlicates understanding, therefore it increases complexity :). There is another name for isomorphic JavaScript - "Universal JavaScript", but in my opinion the word "Universal" is too general. So, in this post, I will use the word "isomorphic" ( even if do not like it :) ).
```
Comment (goatslacker) [inline posts/2015-12-12-isomorphic-react-in-real-life.md]: ismorphic application is not an SPA

**Comment by koorchik** [comment]
Thanks,  in most cases we talk about SPA, but you are right that this is not the same.

Changed to the following:

>An isomorphic JavaScript application is just a JavaScript application that can run both client-side and server-side (in most cases it is a single-page application that can be run on server).

**Comment by goatslacker** [comment]
LGTM

**Comment by koorchik** [comment]
Great. Should I do anything else to make the post approved?

**Comment by zackify** [comment]
Sorry, busy weekend. Will read it over and have it up tonight!

**Pull request merged**